src/style.css

```diff
@@ -1,3 +1,74 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+
+  --active-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.header-logo {
+  display: flex;
+  margin-right: 243px;
```

Comment:
Avoid fixing container sizes, unless it's necessary for the design. It's better to let the content size dictate it. Fixed margins can lead to unexpected results on different screen sizes. Consider making this margin responsive, or using a different method to space elements, like flexbox or grid.